README.md

```
@@ -17,6 +17,7 @@ Moody is a **minimalistic** and modern mood-tracking web application built with
 - [Getting Started](#-getting-started)
 - [How to Use](#-how-to-use)
 - [CI/CD & Docker Automation](#-cicd--docker-automation)
+- [Testing](#-testing)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)